tools/constraints.txt

```diff
@@ -17,6 +17,6 @@ pylint>=2.7.0
 pyenchant>=3.0.0
 test-generator==0.1.1
 py>=1.9.0
-urllib3>=1.26.4
+urllib3==1.26.5
```

Comment:
I don't think to set exact version for the module is good. For some platform, this exact version can be absent and such change is not good for the future.
May be it is better to use markers to differentiate the exact platform.

Reply:
We already use such approach with version 1.26.5 in other components (for example docs).
 now we have a problem with the newest version of urllinb3

Reply:
docs are not a reference for us because no one user needs docs building.
Which tool is using this module? May be we can remove this at all? Alternatively, we can check `ssl` module that seems outdated installed in CI and should be installed explicitly then.

Reply:
I see that the issue is raised on Python 3.8. So let us make this change for this particular version of Python, for others use the newest one. It seems that there is no compiled ssl module on Python 3.8